products/artifactory.md

```diff
@@ -0,0 +1,124 @@
+releasePolicyLink: https://jfrog.com/help/r/jfrog-release-information/artifactory-end-of-life
+releaseDateColumn: true
+releaseColumn: true
+eolColumn: Support
+
+auto:
+-   custom: true
+
+releases:
+-   releaseCycle: "7.59"
+    releaseDate: 2023-05-28
+    eol: 2024-11-28
+    latest: "7.59.11"
+    latestReleaseDate: 2023-06-18
+
+-   releaseCycle: "7.55"
+    releaseDate: 2023-02-28
+    eol: 2024-08-28
+    latest: "7.55.14"
+    latestReleaseDate: 2023-05-23
+
+-   releaseCycle: "7.49"
+    releaseDate: 2022-12-29
+    eol: 2024-06-29
+    latest: "7.49.10"
+    latestReleaseDate: 2023-03-07
+
+-   releaseCycle: "7.47"
+    releaseDate: 2022-12-01
+    eol: 2024-06-01
+    latest: "7.47.16"
+    latestReleaseDate: 2023-01-30
+
+-   releaseCycle: "7.46"
+    releaseDate: 2022-10-11
+    eol: 2024-04-11
+    latest: "7.46.23"
+    latestReleaseDate: 2023-01-31
+
+-   releaseCycle: "7.41"
+    releaseDate: 2022-07-11
+    eol: 2024-01-11
+    latest: "7.41.14"
+    latestReleaseDate: 2022-10-11
+
+-   releaseCycle: "7.39"
+    releaseDate: 2022-06-20
+    eol: 2023-12-20
+    latest: "7.39.10"
+    latestReleaseDate: 2022-08-03
+
+-   releaseCycle: "7.38"
+    releaseDate: 2022-05-08
+    eol: 2023-11-08
+    latest: "7.38.17"
+    latestReleaseDate: 2022-08-11
+
+-   releaseCycle: "7.37"
+    releaseDate: 2022-04-14
+    eol: 2023-10-14
+    latest: "7.37.17"
+    latestReleaseDate: 2022-08-04
+
+-   releaseCycle: "7.35"
+    releaseDate: 2022-03-01
+    eol: 2023-09-01
+    latest: "7.35.2"
+    latestReleaseDate: 2022-03-09
+
+-   releaseCycle: "7.33"
+    releaseDate: 2022-02-03
+    eol: 2023-08-03
+    latest: "7.33.9"
+    latestReleaseDate: 2022-02-07
+
+-   releaseCycle: "7.31"
+    releaseDate: 2022-01-10
+    eol: 2023-07-10
+    latest: "7.31.12"
+    latestReleaseDate: 2022-01-25
+
+-   releaseCycle: "7.29"
+    releaseDate: 2021-01-11
+    eol: 2023-06-05
+    latest: "7.29.9"
+    latestReleaseDate: 2021-01-11
+
+-   releaseCycle: "7.27"
+    releaseDate: 2021-09-30
+    eol: 2023-03-30
+    latest: "7.27.10"
+    latestReleaseDate: 2021-11-07
+
+-   releaseCycle: "7.25"
+    releaseDate: 2021-09-02
+    eol: 2023-03-02
+    latest: "7.25.7"
+    latestReleaseDate: 2021-09-10
+
+-   releaseCycle: "7.24"
+    releaseDate: 2021-09-02
+    eol: 2023-02-11
+    latest: "7.24.7"
+    latestReleaseDate: 2021-12-16
+
+---
+
+> [JFrog Artifactory](https://jfrog.com/artifactory/) is a binary repository manager software
+> designed to store the binary output of the build process for use in distribution and deployment.
+> Artifactory provides support for a number of package formats such as Maven, Conan, Debian APT,
+> NPM, Helm, Ruby, Python, and Docker.
+
+Artifactory is proposed both as a self-hosted solution and as a SaaS solution and not all releases
+are made available to self-hosted customers by JFrog. This page only tracks those releases for
+the commercial editions (self-hosted Artifactory OSS only manage Java binary artifacts).
```

Comment:
What's a commercial edition here? We should use consistent naming (self-hosted v/s SaaS perhaps?) What does upstream use?

Reply:
Simplified a bit the sentence, it should be clearer now.